praetorian_cli/ui/console/commands/context.py

```
@@ -14,11 +14,28 @@ def _cmd_set(self, args):
+        key, value = args[0].lower(), ' '.join(args[1:]).strip().rstrip('.')
         if key == 'account':
+            # Always validate against the real account list
+            if not (hasattr(self, '_account_list') and self._account_list):
+                try:
+                    accounts_list, _ = self.sdk.accounts.list()
+                    self._account_list = [a.get('name', '') for a in (accounts_list or [])]
+                except Exception:
+                    self._account_list = []
+            if self._account_list and value not in self._account_list:
+                self.console.print(f'[error]Unknown account: {value}[/error]')
+                self.console.print(f'[dim]Run "accounts" to see the list, or "switch <#>" to select by number.[/dim]')
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)